CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # NHS.UK frontend Changelog
 
+## 4.0.0 - 19 November 2019
+
+:boom: **Breaking changes**
```

Comment:
We may need to elaborate on this with it being a breaking, i.e whats changed, what people need to do to update, new class names, removed class names etc.